datasets/us/wy/med_exclusions/us_wy_med_exclusions.yaml

```diff
@@ -0,0 +1,40 @@
+title: US Wyoming Medicaid Provider Exclusion List
+entry_point: crawler.py
+prefix: us-medwy
+coverage:
+  frequency: daily
+  start: "2024-09-24"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Medical providers deemed ineligible to participate in Wyoming State's Medicaid program.
+description: |
+  The Wyoming Department of Health maintains a list of providers that have been excluded from
+  the state Medicaid program. Providers listed are not allowed to receive reimbursement for
+  Medicaid services in any capacity.
+publisher:
+  name: Wyoming Department of Health
+  description: >
+    > Wyoming Medicaid is a joint federal and state government program that pays for medical
+    > care for some low-income and medically needy individuals and families.
+  acronym: WDH
+  url: https://health.wyo.gov
+  official: true
+  country: "us"
+ci_test: false
+url: "https://health.wyo.gov/healthcarefin/medicaid/"
+data:
+  url: https://health.wyo.gov/wp-content/uploads/2021/07/2021_07_07_Wyoming-Medicaid-Exclusion-List-PDF_AP.pdf
```

Comment:
it looks like we can crawl up to date links from this page
![image](https://github.com/user-attachments/assets/33c40664-46ab-4f76-865a-42e72faa971e)